src/common/net/Listener.cc

```diff
@@ -30,18 +30,12 @@
 
 namespace hf3fs::net {
 
-static bool checkNicType(std::string_view nic, Address::Type type) {
```

Comment:
inetworkType_ is not get from load (), this fun post "RDMA" to networkType_, this case , NIC only filter by name